lib/ui/vote_answer.dart

```diff
@@ -25,30 +25,105 @@ class _VoteAnswerScreenState extends State<VoteAnswerScreen> {
     'Resposta 6',
   ];
 
-  Widget buildAnswerButton(String resposta) {
-    return OutlinedButton(
-        onPressed: () {
-          votedAnswer = resposta;
-          sendVote();
-        },
-        child: Text(resposta)
-    );
-  }
-
   @override
   Widget build(BuildContext context) {
+    final screenWidth = MediaQuery.of(context).size.width;
+    final appBarHeight = AppBar().preferredSize.height;
+
     return Scaffold(
+      appBar: AppBar(
+        automaticallyImplyLeading: false,
+      ),
       body: SafeArea(
         child: Column(
+          crossAxisAlignment: CrossAxisAlignment.stretch,
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
           children: [
-            const Text('Escolha a melhor resposta'),
-            const Text('Lorem ipsum dolor sit amet, consectetur _____ elit ut aliquam, purus sit amet luctus venenatis, lectus'),
-            for ( var answer in allAnswers) buildAnswerButton(answer),
+            Stack(
+              alignment: AlignmentDirectional.center,
+              children: [
+                LinearProgressIndicator(
+                  minHeight: appBarHeight * 0.73,
+                  color: kPlayer_3,
+                  backgroundColor: kBackgroundDarkGray,
+                ),
+                const Text('Escolha a melhor resposta'),
+              ],
+            ),
+            Container(
+              padding: EdgeInsets.symmetric(
+                horizontal: screenWidth * 0.1,
+              ),
+              decoration: const BoxDecoration(
+                image: DecorationImage(
+                  //TODO: Adicionar imagem de balão
```

Comment:
You can use the  [ChatBalloon](https://github.com/yuumasato/have_you_heard/blob/master/lib/widgets/chat_balloon.dart#L5).
For example: https://github.com/yuumasato/have_you_heard/blob/master/lib/ui/onboarding.dart#L197

Do you plan to do it now or later?

Reply:
Added chatBaloon